### PR TITLE
EFM-1698 Remove "Agent" column from cluster-status output.

### PR DESCRIPTION
**Pull request by EFM-Bobby**
This column was removed ~4 years ago, see EFM-993.

### CODE REVIEW

**Comment by djw-m** [comment]
The column has gone but the surrounding text refers to the status of the agent and without that column it doesn't seem to make sense?

File: product_docs/docs/efm/4/06_monitoring_efm_cluster.mdx [hunk 1]
```diff
@@ -55,11 +55,11 @@ Standby database(s) in sync with primary. It is safe to promote.
 The cluster status section provides an overview of the status of the agents that reside on each node of the cluster:
```
Comment (djw-m) [inline product_docs/docs/efm/4/06_monitoring_efm_cluster.mdx:55]: Specifically, this says the status of the agents, but the table shown doesn't give an agent status, only an agent type. You may be able to infer the agent status from the DB status but....

File: product_docs/docs/efm/4/06_monitoring_efm_cluster.mdx [hunk 2]
```diff
@@ -90,9 +90,9 @@ Standby  172.19.10.2     0/4000638          0/4000638
 If a database is down or if the database was restarted, but the resume command was not yet invoked, the state of the agent that resides on that host is idle. If an agent is idle, the cluster status report includes a summary of the condition of the idle node. For example:
```
Comment (djw-m) [inline product_docs/docs/efm/4/06_monitoring_efm_cluster.mdx:90]:
```suggestion
If a database is down or if the database was restarted, but the resume command was not yet invoked, the state of the agent that resides on that host is idle. If an agent is idle, the cluster status report shows that status in the Agent Type column. For example:
```

File: product_docs/docs/efm/4/06_monitoring_efm_cluster.mdx (hunk 1 above)
Comment (EFM-Bobby) [reply product_docs/docs/efm/4/06_monitoring_efm_cluster.mdx:55]: Correct, the text is just referring to the general set of info about the agent: what type it is, does it have the VIP, whatever else. So I think the existing text is still ok.

File: product_docs/docs/efm/4/06_monitoring_efm_cluster.mdx (hunk 2 above)
Comment (EFM-Bobby) [reply product_docs/docs/efm/4/06_monitoring_efm_cluster.mdx:90]: Good thinking.

**Review by djw-m — APPROVED** [review]
LGTM (concerns addressed)

**Pull request merged**